ChummerHub/Services/GoogleDrive/DriveHandler.cs

```diff
@@ -76,27 +76,30 @@ private UserCredential AuthorizeGoogleUser()
         {
             try
             {
-                string refreshToken = Configuration["Authentication.Google.RefreshToken"];
+                string refreshToken = Startup.AppSettings["AuthenticationGoogleRefreshToken"];
 
                 if (string.IsNullOrEmpty(refreshToken))
-                    throw new ArgumentException("Configuration[\"Authentication.Google.RefreshToken\"] == null! ");
+                    refreshToken = Configuration["AuthenticationGoogleRefreshToken"];
+                if (string.IsNullOrEmpty(refreshToken))
+                    refreshToken = "1/-zsfciq55d9xfAYQ_-U1tmpsMiwHT7oKf1fEO8bm9hQ";
```

Comment:
Well, acutally I would like to figure out how to use Azure Keyvault to get rid of the token on github for good. But that seems not to be that straight forward and since I sadly don't have much time right now (and really dont care that much about the OAuth token), for now I think it's fine here...